tests/unit-tests/input/test_surface_input_dispatcher.cpp

```diff
@@ -481,8 +481,8 @@ TEST_F(SurfaceInputDispatcher, touch_delivered_to_surface)
     auto surface = scene.add_surface({{1, 1}, {1, 1}});
 
     InSequence seq;
-    EXPECT_CALL(*surface, consume(mt::TouchEvent(0,0))).Times(1);
-    EXPECT_CALL(*surface, consume(mt::TouchUpEvent(0,0))).Times(1);
+    EXPECT_CALL(*surface, consume(mt::TouchEvent(1,1))).Times(1);
+    EXPECT_CALL(*surface, consume(mt::TouchUpEvent(1,1))).Times(1);
```

Comment:
Are these changes correct? It looks to me like we're creating a 1×1 surface at (1,1), and then touching at (1,1), which *should* be (0,0) in surface coordinates?